rasa/cli/arguments/train.py

```diff
@@ -46,6 +46,7 @@ def set_train_core_arguments(parser: argparse.ArgumentParser):
 
 def set_train_nlu_arguments(parser: argparse.ArgumentParser):
     add_config_param(parser)
+    add_domain_param(parser, default=None)
```

Comment:
why making the default `None`? `get_validated_path` does a lot of magic and would automatically set it to `None` in case the default path does not exist.

Reply:
So that it's `None` in `rasa train nlu --help`, because we don't want to change the default behaviour.

Reply:
Why don't we wanna join the default behavior? And it doesn't change the default behavior anyway, right?